tsconfig.json

```diff
@@ -3,14 +3,55 @@
         "module": "commonjs",
         "target": "es5",
         "lib": ["es2017", "dom"],
-        "sourceMap": true,
-        "declaration": true,
         "experimentalDecorators": true,
         "downlevelIteration": true,
         "noImplicitReturns": true,
         "pretty": true,
         "skipLibCheck": true,
         "typeRoots": ["node_modules/@0xproject/typescript-typings/types", "node_modules/@types"],
-        "strict": true
-    }
+        "strict": true,
+        // These settings are required for TypeScript project references
+        "composite": true,
+        "declaration": true,
+        "declarationMap": true,
+        "sourceMap": true
+    },
+    // The root of the project is just a list of references and does not contain
+    // any top-level TypeScript code.
+    "include": [],
+    "references": [
+        { "path": "./packages/monorepo-scripts" },
```

Comment:
The order here does not matter, but we will need to remember to add any new packages in the future.

Reply:
Mind starting a Quip doc called "How to add new packages to 0x monorepo"? Feel like there are now some must-do steps:

- Add to `tsconfig.json`
- Add to top-level README after it's published

etc...

Reply:
I added a task to Asana for this: https://app.asana.com/0/684263176955174/798329537896649.